core/src/scss/components/date-stacked/_date-stacked.scss

```diff
@@ -0,0 +1,28 @@
+@charset "UTF-8";
+
+//
+// Date Stacked
+//
+// Abbreviated month stacked on top of numerical day of the month.
+//
+// .su-date-stacked--no-background - Black type on transparent background
+//
+// Markup: ../templates/components/date-stacked/date-stacked.twig
+//
+// Style guide: Components.Date Stacked
+//
+.su-date-stacked {
+  @include modular-spacing('padding', 0);
+  background: $color-black;
+  color: $color-white;
+  text-align: center;
+
+  .su-date-month {
+    @include big-paragraph;
+  }
+
+  .su-date-day {
```

Comment:
Again, BEM naming conventions suggest that this should be named `.su-date-stacked__day`.

Reply:
Thanks!